### PR TITLE
Feature #16 Add styles on the text messages of the fields.

### PR DESCRIPTION
**Pull request by yovanha**
# Description
 - We've added styles to the validation messages for the Sign Up and Log In screen.
 - We've implement a new function to detect if the user is putting its credentials wrong in the Log In screen.
 - We've edited the TextField component to alert the users in the Sign Up screen if the email added already exists.
 - We've edited each component to dissapear the error messages when the user made the correct implementation in both screens.
  
  ## Screenshots 
image

### CODE REVIEW

**Review by DemisRincon — APPROVED** [review]

**Review by cortezroberto — DISMISSED** [review]
Good

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by jkzasori — APPROVED** [review]

File: FlightReservation/src/screens/LogIn.js [hunk 1]
```diff
@@ -9,36 +9,26 @@ import {onGoogleButtonPress} from '../helpers/firebaseSignUp';
...
-          <Span text="loading" />
-        </Text>
-      )}
       <Formik
         validationSchema={logInValidationSchema}
         initialValues={{
-          email: '',
-          password: '',
+          email: 'juan1@example.com',
+          password: 'Juan123%',
```
[the remark this answers is not in the document]
Comment (yovanha) [reply FlightReservation/src/screens/LogIn.js:31]: We implement them to make tests for the user Log In, but I will remove it and put it on my local repository.

**Review by jkzasori — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by cortezroberto — APPROVED** [review]
LGTM

**Pull request merged**